types/comtac/CM-1/uplink.spec.js

```diff
@@ -0,0 +1,86 @@
+const chai = require("chai");
+const validate = require("jsonschema").validate;
+const rewire = require("rewire");
+const fs = require("fs");
+
+const assert = chai.assert;
+
+const script = rewire("./uplink.js");
+let defaultSchema = null;
+let lifecycleSchema = null;
+const consume = script.__get__("consume");
+
+function expectEmit(callback) {
+  script.__set__({
+    emit: callback,
+  });
+}
+
+before(function (done) {
+  fs.readFile(
+    __dirname + "/default.schema.json",
+    "utf8",
+    function (err, fileContents) {
+      if (err) throw err;
+      defaultSchema = JSON.parse(fileContents);
+    }
```

Comment:
split into two befores

Reply:
test works with the new addition in the Bits library